### DISCUSSION TITLE
Parallelism without using dagit

### DISCUSSION THREAD

**Discussion by AmirAktify** [dagster-io/dagster#4361] [Q&A] [answered]
So, is it possible to setup parallelism outside of execution in Dagit, e.g. via calling an `execute_pipeline` in memory in Python? And if so, how? I find that I can't pass a multiprocess executor to `run_config` if I'm using the `execute_pipeline` route.

**Comment by alangenfeld** [comment, marked as answer]
https://docs.dagster.io/concepts/solids-pipelines/pipeline-execution#multiprocessing-execution
